packages/gestalt/src/IconButton.tsx

```diff
@@ -212,12 +256,11 @@ const IconButtonWithForwardRef = forwardRef<HTMLButtonElement, Props>(function I
       type={type === 'submit' ? 'submit' : 'button'}
     >
       <div
-        className={classnames(styles.button, touchableStyles.tapTransition, {
-          [styles.disabled]: disabled,
-          [styles.enabled]: !disabled,
-          [touchableStyles.tapCompress]: !disabled && isTapping,
-        })}
-        style={compressStyle || undefined}
+        className={isInVRExperiment ? vrStyles : divStyles}
+        style={{
+          ...(!isInVRExperiment && compressStyle),
+          ...{ paddingBottom: label ? '8px' : '0px' },
```

Comment:
can we add paddingBottom in CSS as a space token?

Reply:
Added as padding-top on the label class